src/MainView.vala

```diff
@@ -283,6 +283,7 @@ public class DateTime.MainView : Switchboard.SettingsPage {
         change_tz (datetime1.Timezone);
 
         network_time_radio.bind_property ("active", manual_time_box, "sensitive", BindingFlags.INVERT_BOOLEAN | BindingFlags.SYNC_CREATE);
+        auto_time_zone_radio.bind_property ("active", manual_time_zone_radio, "active", BindingFlags.INVERT_BOOLEAN | BindingFlags.SYNC_CREATE);
```

Comment:
We should make sure to bind to the actual source of truth and not to another widget here

Reply:
@danirabbit when I did that , there seems to be a problem where the manual radio shows a checkmark when selected but when you select the auto radio it is "empty" (inconsistent) - this code appears to be working - the auto radio is bound to actual source of truth and the manual radio in turn depends on the auto radio.

Reply:
I mean to say the following line doesn't fully work - the auto radio doesn't get visual checkmark when clicked:

```
time_zone_settings.bind ("automatic-timezone", manual_time_zone_radio, "active", SettingsBindFlags.INVERT_BOOLEAN);
```

Reply:
It looks like when we connect to the network radio being toggled we're not checking if it's active. So that might be interfering with setting the correct selection. The radios are grouped I think we should only need to actually set one and the other should update just because it's the only other option in the group

Reply:
I didn't understand - is the network group affecting the auto-timezone group?

Only the two radios (auto and manual) for timezone are grouped together but without this PR only the auto radio selected would restore when the plug is initially opened but if the `automatic-timezone == false` then the manual radio, though "enabled", appears to not be active.

Hopefully this isn't much effort - could you try the plug (select manual timezone setting, select your city) and observe that when reopening the plug neither radio button appears active - but this PR appears to fix that?

Alternatively, this PR can be closed if a full port to Gtk4 resolves this issue and another issue https://github.com/elementary/switchboard-plug-datetime/issues/119 I just filed.

Reply:
Sorry I'm on mobile right now so I was just trying to scroll through and see why it might be broken

This plugin is already ported fully to gtk4 :)

Reply:
No worries :-) whenever you (or whoever) gets a chance to test/review this PR...

By saying "not fully ported" I meant to point out that there are deprecation warnings with at least the TreeView (city list) and its models and I don't know if that could be the cause of the inconsistent radio button states (shrug) plus the resurfacing of issue #119